Sources/NIOSSL/NIOSSLHandler.swift

```diff
@@ -342,7 +423,7 @@ public class NIOSSLHandler : ChannelInboundHandler, ChannelOutboundHandler, Remo
                 state = .active
                 completeHandshake(context: context)
             }
-        case .unwrapping, .closing, .unwrapped, .closed:
+        case .unwrapping, .closing, .unwrapped, .closed, .inputClosed, .outputClosed:
```

Comment:
I think this is wrong, `.inputClosed` and `.outputClosed` should match `active` here I think.